help/destinations/catalog/data-partners/merkury-enterprise-identity.md

```diff
@@ -0,0 +1,168 @@
+
+### New Account
+
+#### Authenticate to destination
+
+To authenticate to the destination, fill in the required fields and select **Connect to destination**.
+
+To access your bucket on Experience Platform, you need to provide valid values for the following credentials:
+
+| **Credential** | **Description** |
+|---|---|
+|Access key|The access key ID for your bucket. You can retrieve this value from the Merkury team.|
+|Secret key|The secret key ID for your bucket. You can retrieve this value from the Merkury team.|
+|Bucket name|This is your bucket where files will be shared. You can retrieve this value from the Merkury team.|
+
+{style="table-layout:auto"}
+
+![A screenshot of a computer Description automatically generated](../../assets/catalog/data-partners/merkury-identity/media/image4.png)
+
+#### Fill in destination details
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+
+![A screenshot of destination details](../../assets/catalog/data-partners/merkury-identity/media/image6.png)
+
+
+-   **Name (Required)** - The name the destination will be saved under
+-   **Description** - Short explanation of the destination's purpose
+-   **Bucket Name (Required)** - Name of the Amazon S3 bucket set up on S3
+-   **Folder Path (Required)** - If subdirectories in a bucket are used a path must be defined, or '/' to reference the root path.
+-   **File Type** - Select the format Experience Platform should use for the exported files. Consult your Merkury team for the expected file type for your account.
+
+[!NOTE]
+When selecting the CSV option, Delimiter, Quote Character, Escape Character, Empty Value, Null Value, Compression format, and Include manifest file options will be presented, consuly your Merkury team for the appropiate settings for your account.
+
+![image of csv option](../../assets/catalog/data-partners/merkury-identity/media/image8.png)
+
+### Existing Account
+
+Accounts already defined using the Merkury Enterprise Identity destination appear in a list pop-up. When selected, you can see details on the account in the right rail. View the example from the UI, when you navigate to **Destinations** \> **Accounts**;
+
+![A screenshot of destionation account in detaination accounts page](../../assets/catalog/data-partners/merkury-identity/media/image5.png)
+
+
+### Enable alerts
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the
+UI](https://experienceleague.adobe.com/en/docs/experience-platform/destinations/ui/alerts).
+
+When you are finished providing details for your destination connection, select **Next**.
+
+## Activate audiences to this destination
+
+>[!IMPORTANT]
```

Comment:
incorrect note formatting, please update

Reply:
@vgiurgiu, the call out here is for a [!IMPORTANT], I have updated the one [!NOTE] item to match the [!IMPORTANT] changes.  Let me know if this is what you were referring to or not.

Like so:
>[!NOTE]

>When selecting the CSV option....